rsts/deployment/cluster_config/general.rst

```diff
@@ -212,3 +212,31 @@ Any inbound ``CreateExecution`` requests with **[Domain: Production, Project: wi
 Any inbound ``CreateExecution`` requests with **[Domain: Production, Project: widgetmodels]** for any workflow other than ``Demand`` and any launch plan will have a tag value "critical".
 
 All other inbound CreateExecution requests will use the default values specified in the FlyteAdmin config (if any).
+
+
+Configuring K8s Pod using a Default PodTemplate
+------------------------------------------------
+
+FlytePropeller supports configuration of all K8s Pods executed using the Pod plugin. This is accomplished by creating a default PodTemplate. This functionality requires the `default-pod-template-name <https://docs.flyte.org/en/latest/deployment/cluster_config/flytepropeller_config.html#default-pod-template-name-string>`__ configuration option to be set in FlytePropeller.
+
```

Comment:
Can we add something about how this was previously done with one-off configuration options in the [k8s plugin config](https://github.com/flyteorg/flyteplugins/blob/902b902fcf487f30ebb5dbeee3bb14e17eb0ec21/go/tasks/pluginmachinery/flytek8s/config/config.go#L67-L162)? Using a default PodTemplate means all options are configurable without having to update the codebase.